proca/lib/proca/service/email_template.ex

```diff
@@ -109,7 +115,17 @@ defmodule Proca.Service.EmailTemplate do
   end
 
   def compile_string(nil), do: nil
-  def compile_string(m), do: :bbmustache.parse_binary(m)
+
+  def compile_string(m) do
+    try do
+      :bbmustache.parse_binary(m)
+    rescue
+      error ->
+        Sentry.capture_exception(error, stacktrace: __STACKTRACE__)
+        # TODO: return proper error instead of reraising
+        reraise Sentry.CrashError.exception(error.original), __STACKTRACE__
```

Comment:
@tttp Unfortunately I haven't found any easy way to add additional context to original error. So Sentry will capture two exceptions, one with additional context and one without. We can fix it later by returning here proper error